INSTALL.md

```diff
@@ -788,6 +788,11 @@ Build (and install) the FIPS provider
 Don't perform FIPS module run-time checks related to enforcement of security
 parameters such as minimum security strength of keys.
 
+### no-fips-ems_check
+
+Don't perform FIPS module run-time check related to enforcement of EMS
+during TLS1_PRF KDF.
+
```

Comment:
Given this check is disabled by default I do not think we need a build-time configuration option for this functionality.

Reply:
I prefer having the option available like we do for the security checks.

Reply:
But these are on by default. I believe this will only confuse people.

Reply:
I would prefer to keep it consistent, they have different purposes - one strips out the code.

Reply:
I have updated the doc to make it clearer.